src/index.html

```diff
@@ -3,11 +3,508 @@
+                <h4 class="benefits__title benefits__title--one">Futuristic Design</h4>
+                <p class="benefits__text">To give Luna a truly flawless look, we specifically picked aircraft grade aluminum as its material and adopted both three-dimensional stretch-bending technology and a high precision cold forging technique.</p>
+              </div>
+            </div>
+
+            <div class="benefits__box benefits__box--bottom benefits__box--none">
+              <img
+                src="images/icon/Frame-35.svg"
+                alt="img"
+                class="benefits__img animation"
+              >
+
+              <div class="benefits__wrapper">
+                <h4 class="benefits__title benefits__title--two">Tweeter Speaker System</h4>
+                <p class="benefits__text">To deliver a more layered sound performance better than a sole full-range speaker, our team equipped Luna with one more tweeter speaker responsible for high-frequency sound independently.</p>
+              </div>
+            </div>
+
+            <div class="benefits__box benefits__box--none">
+              <img
+                src="images/icon/Frame-36.svg"
+                alt="img"
+                class="benefits__img animation"
+              >
+
+              <div class="benefits__wrapper">
+                <h4 class="benefits__title benefits__title--one">Multiroom System </h4>
+                <p class="benefits__text">Luna is natively compatible with your home Wi-Fi. Set up multiple speakers in different rooms to expand your music experience into the entire house.</p>
+              </div>
+            </div>
+
+            <div class="benefits__box benefits__box--none">
+              <img
+                src="images/icon/Frame-37.svg"
+                alt="img"
+                class="benefits__img animation"
+              >
+
+              <div class="benefits__wrapper">
+                <h4 class="benefits__title benefits__title--two">Intuitive Lighting System</h4>
+                <p class="benefits__text">An intuitive user interface allows you to adjust the hue and saturation of color for lighting that fits any mood and situation.</p>
+              </div>
+            </div>
+
+          </div>
+        </div>
+      </section>
+
+      <section class="presentation section">
+        <div class="container">
+          <div class="grid-container">
+            <div class="presentation__bg">
+              <img src="images/image-2.png" alt="images">
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="about section" id="about">
+        <div class="container">
+          <div class="grid-container">
+            <h2 class="about__mobile">DESIGNED FOR THE FUTURE</h2>
+
+            <div class="about__case">
+              <img
+                src="images/54.png"
+                alt="img"
+                class="about__img general__img animation"
```

Comment:
The same is for general__img class. I recommend you to check typical mistakes with BEM by this [link](https://mate-academy.github.io/fe-program/css/typical-bem-mistakes.html)